### PR TITLE
[front] feat: users can skip the tutorial

### PR DESCRIPTION
**Pull request by OoIwazaruoO**
**related to** #1099

---

This PR has for goal to allow user to skip the tutorial

- [x] add a button to skip the tutorial
- [x] save the user choice to skip the tutorial (at the moment inside the local storage)

skiptutorial

### CODE REVIEW

File: frontend/src/features/comparisonSeries/ComparisonSeries.tsx [hunk 1]
```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
...
+  const skipTutorial = () => {
+    closeDialog();
+
+    setTutorialIsSkipped(true);
+    window.localStorage.setItem('tutorialIsSkipped', 'true');
+  };
+
+  const skipTutorialButton = (
+    <Button
+      sx={{ marginRight: '8px' }}
```
Comment (GresilleSiffle) [inline frontend/src/features/comparisonSeries/ComparisonSeries.tsx:232]: We can simplify the CSS by completely removing the `sx` here.

The button is displayed in a flex container, so we just have to specify a `gap` to define space between all flex items.

File: frontend/src/features/comparisonSeries/ComparisonSeries.tsx [hunk 2]
```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
     setDialogOpen(false);
   };
 
+  const skipTutorial = () => {
+    closeDialog();
+
+    setTutorialIsSkipped(true);
+    window.localStorage.setItem('tutorialIsSkipped', 'true');
```
Comment (GresilleSiffle) [inline frontend/src/features/comparisonSeries/ComparisonSeries.tsx:227]: Let's track this event in Plausible.

We can add the following line at the end of the `skipTutorial` funcction (we also need to update `TRACKED_EVENTS`).

```TypeScript
trackEvent(TRACKED_EVENTS.tutorialSkipped, { props: { step: step } });
```

```TypeScript
export const TRACKED_EVENTS = {
  signup: 'signup',
  tutorial: 'tutorial',
  tutorialSkipped: 'tutorial skipped',
  accountDeleted: 'account deleted',
};
```

File: frontend/src/features/comparisonSeries/ComparisonSeries.tsx [hunk 3]
```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
...
+  };
+
+  const skipTutorialButton = (
+    <Button
+      sx={{ marginRight: '8px' }}
+      color="secondary"
+      variant="contained"
+      onClick={skipTutorial}
+    >
+      {t('tutorial.skip')}
```
Comment (GresilleSiffle) [inline frontend/src/features/comparisonSeries/ComparisonSeries.tsx:237]: We can harmonize the translation key with the rest of the app: `t('tutorial.skipTheTutorial')`

File: frontend/src/features/comparisonSeries/ComparisonSeries.tsx (hunk 1 above)
Comment (GresilleSiffle) [reply frontend/src/features/comparisonSeries/ComparisonSeries.tsx]: Fixed by https://github.com/tournesol-app/tournesol/pull/1470/commits/99f51a1341a29c60a6219babba2bb194f1963c55

File: frontend/src/features/comparisonSeries/ComparisonSeries.tsx (hunk 2 above)
Comment (OoIwazaruoO) [reply frontend/src/features/comparisonSeries/ComparisonSeries.tsx:227]: fixed by > bd1c895

**Review by GresilleSiffle — APPROVED** [review]
Good for me :+1: 

I could have included the skipped state in the Redux store, but it looked a bit overkill? I think I should have done it in fact, to avoid using two different mechanisms and to benefit from the immutability of the store.

**Review by amatissart — APPROVED** [review]

**Pull request merged**